UNR-2577 Docs change to make VS 2017 download steps clearer (#1609)
spatialos/UnrealGDK: SpatialGDK/Documentation/content/get-started/dependencies.md

- The [**DirectX End-User Runtimes (June 2010)**](https://www.microsoft.com/en-us/download/details.aspx?id=8109)

  - You need the DirectX End-User Runtime to run Unreal Engine 4 clients.
  
- <a href="https://visualstudio.microsoft.com/vs/older-downloads/" data-track-link="Clicked VS 2017|product=Docs|platform=Win|label=Win">**Visual Studio 2017**</a>.  

> To download Visual Studio 2017, you must sign into your Microsoft account & sign up to the free [Visual Studio Dev Essentials](https://visualstudio.microsoft.com/dev-essentials/) to access and download the Visual Studio 2017 installer.  

Reviewer: ```suggestion
To access and download the Visual Studio 2017 installer you must sign up for a Microsoft account and sign up to the free [Visual Studio Dev Essentials](https://visualstudio.microsoft.com/dev-essentials/).
```
